modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.scss

```diff
@@ -1,17 +1,17 @@
-@import './AllowedFragments.scss';
 @import 'variables';
 
+@import './AllowedFragments';
```

Comment:
This one was obviously in the wrong place, so I moved it. Also, it had a superfluous ".scss" extension; I created https://github.com/liferay/liferay-npm-tools/issues/391 to track the automate removal of those.